### ISSUE TITLE
Repetitive Link Text

### ISSUE THREAD

**Issue by yvonnetse** [BraftonSupport/braftonium#75]
Fails:
the_content( 
	__( 'Continue reading', 'textdomain' ) 
);

Passes:
the_content( 
	sprintf( 
		__( 'Continue reading%s', 'textdomain' ), 
		'<span class="screen-reader-text">  '.get_the_title().'</span>' 
	) 
);

**Comment by yvonnetse** [comment]
Fixed!
